src/openms/source/CHEMISTRY/ModificationsDB.cpp

```diff
@@ -56,6 +56,18 @@ namespace OpenMS
     readFromOBOFile("CHEMISTRY/PSI-MOD.obo");
   }
 
+  ModificationsDB::ModificationsDB(OpenMS::String unimod_file, OpenMS::String psimod_file)
```

Comment:
why not
```
ModificationsDB::ModificationsDB(OpenMS::String unimod_file = "CHEMISTRY/unimod.xml" , OpenMS::String psimod_file = "CHEMISTRY/PSI-MOD.obo")
{
  readFromUnimodXMLFile(unimod_file);
  readFromOBOFile(psimod_file);
}
```

and be done with it, one function for both use cases and even transparent what the default options are (unlike now)